### PR TITLE
Django Girls presentation draft

### PR DESCRIPTION
**Pull request by w1stler**

### CODE REVIEW

**Comment by magul** [comment]
Looks fine for me, but it's only draft right now. Do You want me to merge it or will You work on it further?

**Comment by w1stler** [comment]
I'll update it today and then we can merge it.

File: django_girls.html [hunk 1]
```diff
@@ -51,18 +50,19 @@
...
                             </section>
                     </section>
                 </section>
                 <section>
-                    <h1>Questions?</h1>
-                    <h5 class="fragment">Thank you for your attention</h2>
-                    <h5 class="fragment"><a href="mailto:wojciech.patelka@gmail.com">wojciech.patelka@gmail.com</a></h2>
+                    <h1>Pytania?</h1>
+                    <h5 class="fragment">Dziękuję za uwagę.</h2>
+                    <h5 class="fragment"><a href="mailto:wojciech.patelka@gmail.com">hello@codeforpoznan.pl</a></h2>
```
Comment (magul) [inline django_girls.html]: I doubt, that anybody will see this presentation online, but still `mailto:wojciech.patelka@gmail.com`?

File: django_girls.html [hunk 2]
```diff
@@ -25,8 +25,7 @@
         <div class="reveal">
             <div class="slides">
                 <section>
-                    <h1>Code for Poznan </h1>
```
Comment (magul) [inline django_girls.html]: I was rather thinking about moving logo here and removing secons slide.

Something like this:
```
                <section>
                    <section>
                        <h1>
                            <img src="img/logo_code_for_poznan.png" style="background:none; border:none; box-shadow:none; "/>
                            <h3>Django Girls</h3>
                            <h4>24.06.2017, Poznań</h4>
                        </h1>
                    </section>
                        <section>
                        <h2>Misja</h2>
                        <p class="fragment">Technologiczne rozwiązania społecznych problemów</p>
                        </section>
                </section>
```

File: django_girls.html [hunk 3]
```diff
@@ -51,18 +50,19 @@
...
                 </section>
                 <section>
-                    <h1>Questions?</h1>
-                    <h5 class="fragment">Thank you for your attention</h2>
-                    <h5 class="fragment"><a href="mailto:wojciech.patelka@gmail.com">wojciech.patelka@gmail.com</a></h2>
+                    <h1>Pytania?</h1>
+                    <h5 class="fragment">Dziękuję za uwagę.</h2>
+                    <h5 class="fragment"><a href="mailto:wojciech.patelka@gmail.com">hello@codeforpoznan.pl</a></h2>
                     <h5 class="fragment"><a href="https://github.com/CodeForPoznan">https://github.com/CodeForPoznan</a></h3>
+                    <h5 class="fragment"><a href="www.codeforpoznan.pl">codeforpoznan.pl</a>
```
Comment (magul) [inline django_girls.html]: if we use https in github address we should use http here also (in body and in href)

**Pull request merged**